package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "checkup",
+  "version": "0.0.1",
+  "main": "index.js",
+  "repository": "https://github.com/checkupjs/checkup.git",
+  "author": "Steve Calvert <steve.calvert@gmail.com>",
+  "license": "MIT",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "test": "yarn workspaces run test",
+    "lint": "eslint packages/* --ext .ts"
```

I think you should probably do:

```suggestion
    "lint": "eslint . --cache --ext .ts"
```

Specifically, we want to cache (makes repeated runs much faster) and we should be able to do `eslint .`, that way we also ensure any top level scripts get linted (like `.eslintrc.js`). 

You may need to tweak `.eslintignore` do enable this.

---

Fixed. This was an easy change, and a good one.